src/login/logind-dbus.c

```
@@ -1668,6 +1668,7 @@ static int method_set_user_linger(sd_bus_message *message, void *userdata, sd_bu
                         return r;
 
                 if (manager_add_user_by_uid(m, uid, &u) >= 0) {
+                        user_send_changed(u, "Linger");
```

please cast to void, to tell static checkers you knowingly ignore errors here

---

Done, also noticed a few others, send https://github.com/systemd/systemd/pull/39254 to handle those.